database/buildSchema/50_views_functions_triggers.sql

```diff
@@ -1126,6 +1126,21 @@ $$
 LANGUAGE sql
 STABLE;
 
+-- Checks if TOTAL of assigned questions are approved/agreed - checking for completeness of assignable questions
+CREATE FUNCTION public.approved_assigned_questions_count (app_id int, stage_id int, level_number int)
+    RETURNS bigint
+    AS $$
+    SELECT
+        COUNT(*)
+    FROM
+        assigned_questions (app_id, stage_id, level_number) aq
+WHERE
+    aq.decision = 'APPROVE'
+    OR aq.decision = 'AGREE'
+$$
+LANGUAGE sql
+STABLE;
+
```

Comment:
Is this function used anywhere? It's not in this PR?

Reply:
Yes, it's used on the Front-end PR & on the Configuration I added to the Template action `IncrementStage`

Reply:
Oh, so there's a function that is *only* called by a config query? 

I'm not sure if that's a good idea or not. Maybe put a comment mentioning that, otherwise someone like me is going to come along in a few months and go "this function's not being used anywhere, let's delete it"!

Reply:
No problem - Adding a comment now